config/error/error_constant.go

```diff
@@ -0,0 +1,19 @@
+package error
+
+const (
+	ChangeToRootDirFailure   = "Failed to change to root directory: "
+	ConnectToDatabaseFailure = "Failed to connect database: "
+	DeleteRedisJobFailure    = "Failed to delete redis job: "
+	EnqueueJobFailure        = "Failed to enqueue job: "
+	HashPasswordFailure      = "Failed to hash password: "
```

Comment:
While all the other error messages make sense in this file as they seem to apply to the whole application, I found it weird to have "HashPasswordFailure" as I thought this would be in a user model or service. 

However, I found out  `bcrypt.GenerateFromPassword` is used in many many places. 

![image](https://user-images.githubusercontent.com/696529/107614326-ad0d3300-6c7c-11eb-95a6-55fcfaa3dda6.png)

Just as the dependency for log was imported in many places, there is the same issue with `golang.org/x/crypto/bcrypt`. Instead, there should be an internal package to abstract this and it should also host this error message to be specific to that package.

Not a blocker for this PR but this is an idea for further improvements. I also wanted to point it out so that you don't do the same mistake in the future. Do not sprinkle dependencies all over the place as this results in tight coupling and harder to debug/maintain applications.

Reply:
Thanks for explanation.
I fixed this in https://github.com/gutakk/go-google-scraper/pull/80.